static/bidder-info/trustedstack.yaml

```diff
@@ -0,0 +1,24 @@
+endpoint: "https://prebid-adapter.trustedstack.com/rtb/pb/trustedstacks2s"
+extra_info: "https://trustedstack.golang.pbs.com"
+maintainer:
+  email: "product@trustedstack.com"
```

Comment:
@product-trustedstack prebid team has sent an email to above specified address inorder to verify its correctness. Requesting to reply back with received message on email thread.

Reply:
Done.

Reply:
received email from `trustedstack`